cookbooks/swift/recipes/setup.rb

```diff
@@ -87,9 +87,8 @@
 # no-no packages (PIP is the bomb, system packages are OLD SKOOL)
 unrequired_packages = [
   "python-requests",  "python-six", "python-urllib3",
-  "python-pbr", "python-pip",
-  "python3-requests",  "python3-six", "python3-urllib3",
-  "python3-pbr", "python3-pip",
+  "python-pbr", "python3-requests",  "python3-six", "python3-urllib3",
+  "python3-pbr",
```

Comment:
I think our py3 strategy should be something like: "always py3 out of the gate, with optionally attempting to support testing on py2"